src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -286,5 +287,21 @@ public boolean equals(Object other) {
                     && getInterviewStatus().equals(e.getInterviewStatus())
                     && getAvailability().equals(e.getAvailability());
         }
+
+
+        /**
+         * Allows the {@code InterviewStatus} to be triggered by ApplicationStatus.
+         */
+        public void triggerInterviewStatus(ApplicationStatus applicationStatus) {
+            String acceptedTrigger = "Accepted";
+            String rejectedTrigger = "Rejected";
+            String executedTrigger = "Completed";
```

Comment:
Nice! Just a minor minor minor suggestion for future, maybe could add the world "final" for defensive programming.

P.S. The word trigger gives me PTSD for 2102 😄

Reply:
HAHA yeah as I was writing this I was thinking about 2102 too. 

I decided to go along with 
```
String acceptedTrigger = ApplicationStatus.ACCEPTED_STATUS;
String rejectedTrigger = ApplicationStatus.REJECTED_STATUS;
String executedTrigger = InterviewStatus.COMPLETED;
```

I think this will help me in the future if we want to add/remove/edit triggers in this method. Hope this suggestion works for you and @domlimm too!

Reply:
Sorry my bad, I merged the changes @domlimm  suggested into github and it conflicted with my local files. Hence now the methods will now be 

```
      if (getApplicationStatus().isPresent()) {
                if (applicationStatus.toString().equals(ACCEPTED_STATUS)
                        || applicationStatus.toString().equals(REJECTED_STATUS)) {
                    setInterviewStatus(new InterviewStatus(COMPLETED));
                }
            }
```

Reply:
Great! 👍🏻